src/coreclr/vm/eventing/eventpipe/ds-rt-coreclr.h

```diff
@@ -201,10 +205,12 @@ ds_rt_generate_core_dump (DiagnosticsDumpCommandId commandId, DiagnosticsGenerat
  			// For the old commmand, this payload field is a bool of whether to enable logging
  			flags = flags != 0 ? GenerateDumpFlagsLoggingEnabled : 0;
 		}
-		if (GenerateDump (reinterpret_cast<LPCWSTR>(ds_generate_core_dump_command_payload_get_dump_name (payload)),
-			static_cast<int32_t>(ds_generate_core_dump_command_payload_get_dump_type (payload)),
-			flags))
+        LPCWSTR dumpName = reinterpret_cast<LPCWSTR>(ds_generate_core_dump_command_payload_get_dump_name (payload));
```

Comment:
Formatting.

Reply:
Looks like tabs are used in these files instead of spaces. I thought the runtime repo convention was spaces?